EventHubForwarder/README.md

```diff
@@ -20,7 +20,7 @@ Parameters that can be configured in your Azure Resource Manager Template
 
 | Parameter  | Required | Default Value | Description
 |---|---|---|---|
-| New Relic License Key  | yes | `none` | Your New Relic Insights [insert key](https://docs.newrelic.com/docs/insights/insights-api/get-data/query-insights-event-data-api#register) |
+| New Relic License Key  | yes | `none` | Your New Relic [License_key](https://docs.newrelic.com/docs/logs/forward-logs/azure-log-forwarding/#azure-eventhub) can be created via ARM template. |
```

Comment:
Please point this link to https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/ instead.

Reply:
Resubmitted.